src/nyc_trees/apps/event/views.py

```diff
@@ -429,3 +445,26 @@ def increase_rsvp_limit(request, event_slug):
     return {
         'max_attendees': event.max_attendees
     }
+
+
+@transaction.atomic
+def event_email_unsubscribe(request, event_slug, token):
```

Comment:
I think there needs to be a test in here somewhere to confirm that the username coming back with the token is the user from the current session. If I click a valid unsubscribe link of another user while logged in as another, it unsubscribes them.